src/base/net/downloadmanager.cpp

```diff
@@ -123,8 +123,10 @@ namespace
 
         // Spoof HTTP Referer to allow adding torrent link from Torcache/KickAssTorrents
         request.setRawHeader("Referer", request.url().toEncoded().data());
-        // Accept gzip
+#ifdef QT_NO_COMPRESS
+        // Manually decompress with gzip
```

Comment:
This comment is confusing... either keep the original or remove it.
In addition, there should be comments explaining why `#ifdef QT_NO_COMPRESS` is related to the code here.